### PR TITLE
Edit issue

### PR DESCRIPTION
**Pull request by seth-bradshaw**
Finished the EditIssue component

### CODE REVIEW

**Review by Wesley-Ryan — APPROVED** [review]
Looks good

File: co-make/README.md [hunk 1]
```diff
@@ -1,68 +0,0 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (tarmes) [inline co-make/README.md:1]: This was somehow deleted, looks like it needs replaced, and then push to branch again before we'll merge

**Review by tarmes — CHANGES_REQUESTED** [review]
the README was somehow deleted and needs replaced before we can merge it

File: co-make/README.md (hunk 1 above)
Comment (tarmes) [reply co-make/README.md:1]: no worries we still have the readme on wes and my branches so it will get replaced

**Pull request merged**